src/app/games/conspirateurs/tests/ConspirateursMinimax.spec.ts

```diff
@@ -1,242 +1,26 @@
-            ], 60);
-            // Here, A is stuck between two occupied shelters
-            const weakState: ConspirateursState = new ConspirateursState([
-                [_, X, O, X, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
-            ], 60);
-            RulesUtils.expectSecondStateToBeBetterThanFirstFor(heuristic,
-                                                               weakState, MGPOptional.empty(),
-                                                               strongState, MGPOptional.empty(),
-                                                               Player.ZERO);
-        });
-    });
-});
+import { AIDepthLimitOptions } from 'src/app/jscaip/AI';
+import { ConspirateursNode, ConspirateursRules } from '../ConspirateursRules';
+import { ConspirateursOrderedMoveGenerator } from '../ConspirateursOrderedMoveGenerator';
+import { ConspirateursHeuristic } from '../ConspirateursHeuristic';
+import { Minimax } from 'src/app/jscaip/Minimax';
+import { ConspirateursState } from '../ConspirateursState';
 
 describe('ConspirateursMinimax', () => {
 
     let rules: ConspirateursRules;
-    let minimax: ConspirateursMinimax;
+    let minimax: Minimax<ConspirateursMove, ConspirateursState>;
     const minimaxOptions: AIDepthLimitOptions = { name: 'Level 1', maxDepth: 1 };
 
     beforeEach(() => {
         rules = ConspirateursRules.get();
-        minimax = new ConspirateursMinimax();
+        minimax = new Minimax('Jump',
+                              ConspirateursRules.get(),
+                              new ConspirateursHeuristic(),
+                              new ConspirateursOrderedMoveGenerator());
     });
+    // TODO FOR REVIEW: que fait on de ce test ? On le généralise sur TOUS les composants ?
+    // TODO FOR REVIEW: genre on s'assure que chaque minimax déclaré doit pouvoir faire X tours contre lui-même
```

Comment:
Allez, faisons !

Reply:
Je propose néanmoins de le faire en ticket : [W] [easy] Minimax tests: ensure all minimaxes can play up to X turns against themselves.
Description: see how it is already done in ConspirateursMinimax